File: Assets/VirtualCity/ProtoDefine/ReqLoginMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]



/**
 * 请求－账号登录
 */
public class ReqLoginMessage {
	
	/** 账号ID */
[ProtoMember(1)]
	public long? accountId;
	/**手机号码*/
[ProtoMember(2)]
	public  string phone;
[ProtoMember(3)]
	public byte[] content;


	public long? getAccountId() {
		return accountId;
	}

	public void setAccountId(long? accounteId) {
		this.accountId = accounteId;
	}


	public string getPhone() {
		return phone;
	}

	public void setPhone(string phone) {
		this.phone = phone;
	}

	public byte[] getContent() {
		return content;
	}

	public void setContent(byte[] content) {
		this.content = content;
	}

	public string tostring() {
		return "ReqChatLoginMessage{" +
				"accountId=" + accountId +
				", phone='" + phone + '\'' +
				'}';
	}

}
}

File: Assets/VirtualCity/ProtoDefine/Java/login/ReqLoginMessage.java
package com.kingston.jforgame.server.game.login.message;

import com.baidu.bjf.remoting.protobuf.FieldType;
import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.entity.master.Account;
import com.kingston.jforgame.server.game.login.LoginDataPool;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

import java.util.Objects;

/**
 * 请求－账号登录
 * @author kingston
 */
@MessageMeta(module=Modules.LOGIN, cmd=LoginDataPool.REQ_LOGIN)
public class ReqLoginMessage extends Message {
	
	/** 账号ID */
	@Protobuf(order = 1)
	private Long accountId;
	/**手机号码*/
	@Protobuf(order = 2)
	private  String phone;
	@Protobuf(order = 3)
	private byte[] content;


	public Long getAccountId() {
		return accountId;
	}

	public void setAccountId(Long accounteId) {
		this.accountId = accounteId;
	}


	public String getPhone() {
		return phone;
	}

	public void setPhone(String phone) {
		this.phone = phone;
	}

	public byte[] getContent() {
		return content;
	}

	public void setContent(byte[] content) {
		this.content = content;
	}

	@Override
	public String toString() {
		return "ReqChatLoginMessage{" +
				"accountId=" + accountId +
				", phone='" + phone + '\'' +
				'}';
	}

}
